src/actions.test.js

```diff
@@ -0,0 +1,12 @@
+import * as actions from './actions'
+
+describe('actions', () => {
```

Comment:
I could be wrong about this, but I _think_ Jest conventions are to have tests in a `__test__` folder. So I'd have the code in `src/actions.js` but the test in `__tests__/actions.test.js`

Reply:
This makes it incredibly easy for this review though, as the test files are next to the code 😉

Reply:
I think the original structure of the create react app has App.js and
App.test.js in /src, but I could be wrong. I wasn’t sure. It can at least
see and run the tests in this structure. I wasn’t sure if all the
components should be in their own folder or not

Reply:
Oh interesting! I prefer it with the code, but I haven't seen it like that in the wild before 😄 

My actual favourite is Rust where the unit tests are _within_ the files 💥